### PR TITLE
[ART-6667] only create defined advisory

### PR DESCRIPTION
**Pull request by Ximinhan**
ref: https://issues.redhat.com/browse/ART-6667
only create placeholder advisory defined in assembly

### CODE REVIEW

**Review by joepvd — COMMENTED** [review]
Getting confused while reading all the chained `if`s. 
Can't we have:
- set `impetus = 'RHEA` if GA, `RHBA` otherwise
- and loop through the advisories in the assembly definition, and create 'em if not exist? 

Or what is the real reason for all this logic?

**Comment by Ximinhan** [comment]
> Getting confused while reading all the chained `if`s. Can't we have:
> 
> * set `impetus = 'RHEA` if GA, `RHBA` otherwise
> * and loop through the advisories in the assembly definition, and create 'em if not exist?
> 
> Or what is the real reason for all this logic?

I guess the previous approach is caused by each advisory needing different args for creating

File: pyartcd/pyartcd/pipelines/prepare_release.py [hunk 1]
```diff
@@ -146,39 +146,13 @@ async def run(self):
...
-                    advisories["metadata"] = self.create_advisory("RHBA", "image", "metadata")
-                else:
-                    _LOGGER.info("Reusing existing metadata advisory %s", advisories["metadata"])
-            # microshift advisory is present since 4.12
-            if self.release_version >= (4, 12):
-                if advisories.get("microshift", 0) <= 0:
-                    advisories["microshift"] = self.create_advisory("RHBA", "rpm", "microshift")
-                else:
-                    _LOGGER.info("Reusing existing microshift advisory %s", advisories["microshift"])
+            is_ga = True if self.release_version[2] == 0 else False
```
Comment (vfreex) [inline pyartcd/pyartcd/pipelines/prepare_release.py:149]:
```suggestion
            is_ga = self.release_version[2] == 0
```

File: pyartcd/pyartcd/pipelines/prepare_release.py [hunk 2]
```diff
@@ -146,39 +146,13 @@ async def run(self):
...
+            for ad in advisories:
+                if advisories[ad] < 0:
+                    if ad == "microshfit":
+                        advisories[ad] = self.create_advisory("RHBA", "rpm", ad)
```
Comment (vfreex) [inline pyartcd/pyartcd/pipelines/prepare_release.py:153]: RHEA for GA as well?

**Review by vfreex — APPROVED** [review]
LGTM

**Pull request merged**